### DISCUSSION TITLE
Does someone knows from where is this outfit/spritepack?

### DISCUSSION THREAD

**Discussion by ChloeRaven** [Monika-After-Story/MonikaModDev#7978] [general - spritepacks]
![7rw2pxd4d1r71](https://user-images.githubusercontent.com/73433526/135731773-901a4e4c-a10a-4767-abae-2e5f81c5c020.png)

**Comment by Capn-Alex** [comment]
I downloaded it from the link in [this comment.](https://www.reddit.com/r/MASFandom/comments/pv3zbi/monika_and_her_new_pj/he8ux3f/) Enjoy!
